src/applications/disability-benefits/all-claims/utils.jsx

```diff
@@ -267,6 +268,38 @@ export function transformProviderFacilities(providerFacilities) {
   }));
 }
 
+export function concatIncidentLocationString(incidentLocation) {
+  return [
+    incidentLocation.city,
+    incidentLocation.state,
+    incidentLocation.country,
+    incidentLocation.additionalDetails,
+  ]
+    .filter(locationField => locationField && locationField.length > 0)
+    .join(', ');
+}
+
+export function getFlatIncidentKeys() {
+  const incidentKeys = [];
+
+  for (let i = 0; i < PTSD_INCIDENT_ITERATION; i++) {
+    incidentKeys.push(`incident${i}`);
```

Comment:
🤔 would it make sense to pull out the `'incident'` string into `constants` as well? What I'm afraid of is changing the key at a later time (say, it gets changed to `ptsdIncidentX`) and we miss changing it here. Could be overkill, but on the other hand you could have all the pages with that schema pull from that constant, as well as any place this ref is needed in the transformers. Thoughts?

Reply:
I understand the concern, and I'd like to address that in a separate PR.